crates/context/src/lib.rs

```diff
@@ -51,7 +51,7 @@ use config::ContextConfig;
 pub struct ContextManager {
     store: Store,
     client_config: ContextConfigClientConfig,
-    config_client: ContextConfigClient<RelayOrNearTransport>,
+    config_client: ContextConfigClient<RelayOrNearOrStarknetTransport>,
```

Comment:
we can rename this to `AnyTransport`

```suggestion
    config_client: ContextConfigClient<AnyTransport>,
```

Reply:
Updated